.github/workflows/windows.yml

```diff
@@ -31,4 +31,4 @@ jobs:
           gcc --version
           $env:ZZZ_SKIP_WINDOWS_SERVER_VERSION_CHECK_NOT_SUPPORTED_IN_PRODUCTION = 'true'
           $packages=go list .\... | Where-Object {$_ -NotMatch 'vendor'}
-          go test -v -tags unit -timeout=40s $packages
+          go test -v -tags unit -timeout=3m $packages
```

Comment:
Thats a big jump. What happened? Similar for below

Reply:
I added a couple tests for polling, and the wait for that is a minute. A couple tests can probably be clubbed to decrease this increase, though there will be some increase still

Reply:
could we overwrite the wait time to a smaller value in test? example:
source:
```
const defaultXXXTimeout = time.Minute
var xxxTimeout = defaultXXXTimeout
```
beginning of test:
```
xxxTimeout = time.Second
defer func() { xxxTimeout = defaultXXXTimeout }()
```

Reply:
+1, lets make sure unit tests are as quick as possible :)